### ISSUE TITLE
FSharpx.Extras 2.3 nuget pulls in SourceLink packages

### ISSUE THREAD

**Issue by marklam** [fsprojects/FSharpx.Extras#372]
### Description

Two SourceLink packages are being pulled in as dependencies, which breaks the build if the solution is not in a git repository
```
SourceLink.Create.CommandLine (>= 2.8.0) 
SourceLink.Embed.PaketFiles (>= 2.8.0)
```

Building the solution in TeamCity from a Mercurial repository fails with the errors:
```
[22:03:20]	[BuildDebug] MSBuild: task: X:\BuildAgent\work\xxx\xxx.sln (28s)
[22:03:24]	[Step 2/2]   fatal: not a git repository (or any of the parent directories): .git
[22:03:24]	[Step 2/2] C:\Users\xxx\.nuget\packages\sourcelink.create.commandline\2.8.3\build\SourceLink.Create.CommandLine.targets(21,5): error MSB3073: The command "git rev-parse --show-toplevel" exited with code 128.
[22:03:24]	[Step 2/2]   fatal: not a git repository (or any of the parent directories): .git
```

### Expected behavior

Presumably these packages shouldn't be included as dependencies

**Comment by panesofglass** [comment]
Agreed. Per the documentation, adding sourcelink should be as simple as adding some attributes to the `Directory.Build.props`.

**Comment by gdziadkiewicz** [comment]
This has been fixed in version 2.3.1
